ckanext/qdes_schema/validators.py

```diff
@@ -305,6 +309,13 @@ def qdes_iso_8601_durations(key, flattened_data, errors, context):
     if has_error:
         raise toolkit.Invalid('The value in each field needs to be positive number.')
 
+    # Validate the pattern.
+    result_pattern = re.split("^P(?=\d+[YMWD])(\d+Y)?(\d+M)?(\d+W)?(\d+D)?(T(?=\d+[HMS])(\d+H)?(\d+M)?(\d+S)?)?$", flattened_data[key])
+    log.error("result_pattern")
```

Comment:
Are these debugging logs?
If they are supposed to be error logs should they be moved to inside the if statement if the result_patten is incorrect